hbw/app/javascript/packs/hbw/components/form/select_table.js.jsx

```diff
@@ -161,9 +161,26 @@ modulejs.define('HBWFormSelectTable',
         this.controlValidationTooltip(this.isValid());
       },
 
+      onClickMulti (event) {
+        if (this.props.params.editable === false) {
```

Comment:
Stay DRY in such simple cases, please

Reply:
General logic is pretty much the same as in ```onClick```.
You check ```editable```, grab some new values and then set new state, validation state and control validation tooltip.